src/main/java/wolox/training/practicae/IBookDao.java

```diff
@@ -1,9 +1,13 @@
 package wolox.training.practicae;
```

Comment:
Noté que te quedó mal la organización de los packages del proyecto: 
- Los DAO irían en repositories
- Los Controllers en controllers

Reply:
corregido